packages/jest-reporters/package.json

```diff
@@ -33,11 +33,13 @@
     "jest-worker": "^28.0.2",
     "slash": "^3.0.0",
     "string-length": "^4.0.1",
+    "strip-ansi": "^6.0.0",
```

Comment:
Noticed that `GitHubActionsReporter` needs it:

https://github.com/facebook/jest/blob/c8c32d3704469557ecfc3d4a9e81002f11e2075d/packages/jest-reporters/src/GitHubActionsReporter.ts#L8